### ISSUE TITLE
Missing file

### ISSUE THREAD

**Issue by dm13dv1b** [OnePlusOSS/android_kernel_oneplus_msm8994#7]
Upon first try to make I got this error:
In file included from include/linux/compiler.h:54:0,
                 from include/uapi/linux/stddef.h:1,
                 from include/linux/stddef.h:4,
                 from /home/happiness/android-kernel/android_kernel_oneplus_msm8994/include/uapi/linux/posix_types.h:4,
                 from include/uapi/linux/types.h:13,
                 from include/linux/types.h:5,
                 from include/linux/mod_devicetable.h:11,
                 from scripts/mod/devicetable-offsets.c:2:
include/linux/compiler-gcc.h:103:30: fatal error: linux/compiler-gcc5.h: No such file or directory
compilation terminated.
There is no compiler-gcc5.h

**Comment by xboxfanj** [comment]
This kernel supports the 4.8 GCC compiler, just as most if not all release kernels for Android 5.1 do. It's not made to build with 5.x, so if you want it to, you'll need to make the proper modifications. Otherwise, use 4.8.

**Comment by dm13dv1b** [comment]
Yes, I know,  that is why I am using 4.9 without other problems... But the file is still missing even if I compile it with 4.8.
